### PR TITLE
Hotfix396 new gdal bug

### PR DESCRIPTION
**Pull request by akorosov**

### CODE REVIEW

**Comment by akorosov** [comment]
@mortenwh, can you check this PR? Then Nansat will not fail on new GDAL. And we maybw can make a new minor release (merge develop into master).

**Comment by mortenwh** [comment]
Coverage decreased... Can you check coveralls, and especially https://coveralls.io/builds/21469959/source?filename=nansat/vrt.py to test at least the new lines (1244 and 1252?)?

File: README.rst [hunk 1]
```diff
@@ -129,8 +129,6 @@ Development is supported by the Research Council of Norway as a part of
 
 .. |Build Status| image:: https://travis-ci.org/nansencenter/nansat.svg?branch=master
    :target: https://travis-ci.org/nansencenter/nansat
-.. |AppVeyor Status| image:: https://ci.appveyor.com/api/projects/status/la50x7l2yy4d9ljr/branch/master?svg=true
```
Comment (mortenwh) [inline README.rst]: Was this used for testing under windows, which we agreed to stop supporting?

File: nansat/vrt.py [hunk 2]
```diff
@@ -1235,6 +1235,11 @@ def get_shifted_vrt(self, shift_degree):
         # Copy self into self.vrt
         shift_vrt = self.get_super_vrt()
 
+        # cancel shift if not longlat projection
+        projection = self.get_projection()
+        if projection[1] != 'dataset' or 'AUTHORITY["EPSG","4326"]]' not in projection[0]:
+            return shift_vrt
```
Comment (mortenwh) [inline nansat/vrt.py]: this is tested right? Seems to be from the coveralls report but the total coverage has decreased.

File: README.rst (hunk 1 above)
Comment (akorosov) [reply README.rst]: Yes

File: nansat/vrt.py (hunk 2 above)
Comment (akorosov) [reply nansat/vrt.py]: This one is tested. But some older lines were not. I've changed the tests to cover some other lines as well.

**Comment by akorosov** [comment]
Now coverage has increased ;)

**Comment by akorosov** [comment]
Also added py37 to travis. Tests pass on my PC.

File: nansat/vrt.py (hunk 2 above)
Comment (mortenwh) [reply nansat/vrt.py]: Great :)

**Comment by akorosov** [comment]
I removed the last commit and forced push.
Apparently travis is not yet ready for py37

**Comment by mortenwh** [comment]
@akorosov rebase and merge will cause conflict. It is ok with a regular merge, right?

**Pull request merged**